streams/src/test/java/org/apache/kafka/streams/processor/internals/TaskManagerTest.java

```diff
@@ -3112,6 +3122,26 @@ void setChangelogOffsets(final Map<TopicPartition, Long> changelogOffsets) {
             return changelogOffsets;
         }
 
+        @Override
+        public Map<TopicPartition, Long> committedOffsets() {
+            return Collections.emptyMap();
+        }
+
+        @Override
+        public Map<TopicPartition, Long> highWaterMark() {
+            return Collections.emptyMap();
+        }
+
+        @Override
+        public Optional<Long> timeCurrentIdlingStarted() {
+            return Optional.empty();
+        }
+
+        @Override
+        public void updateCommittedOffsets(final TopicPartition topicPartition, final Long offset) {
```

Comment:
Could we add a test case for `updateCommittedOffsets`?

Reply:
Since this is a testing implementation of Task I just added some checks to make sure the calls are valid, but I think that is sufficient.